job-server/src/spark.jobserver/WebApi.scala

```diff
@@ -27,16 +28,63 @@ import org.apache.shiro.config.IniSecurityManagerFactory
 import org.apache.shiro.mgt.SecurityManager
 import org.apache.shiro.SecurityUtils
 
+object WebApi {
+  val StatusKey = "status"
```

Comment:
moved code from WebApi class into object so that it can be accessed from elsewhere